functions/github/webhooks.js

```diff
@@ -0,0 +1,154 @@
+const {
+  getReleaseIdFromBranch,
+  updateCheckRunStatus,
+} = require("../database/database.js");
+const {
+  syncReleaseState,
+} = require("../handlers/handlers.js");
+const {
+  verifySignature,
+} = require("./github.js");
+const {Octokit} = require("@octokit/rest");
+const {error, log} = require("firebase-functions/logger");
+const {defineSecret} = require("firebase-functions/params");
+const GITHUB_WEBHOOK_SECRET = defineSecret("GITHUB_WEBHOOK_SECRET");
+const GITHUB_TOKEN = defineSecret("GITHUB_TOKEN");
```

Comment:
I notice you pull in the secret here, and also in index.js in reference to the functions here

Is this an artifact of how functions/secrets work?

Reply:
Yeah, I think I have to define them both for the definition of the handler in `index.js`, and in the function itself when I actually use the token.